x-pack/plugin/esql/qa/testFixtures/src/main/resources/hash.csv-spec

```diff
@@ -0,0 +1,17 @@
+md5
+required_capability: hash_function
+
+ROW value = "test" | EVAL hash = hash("md5", value);
+
+value:keyword | hash:keyword
+test          | 098f6bcd4621d373cade4e832627b4f6
+;
+
+sha256
+required_capability: hash_function
+
+ROW value = "test" | EVAL hash = hash("sha256", value);
+
+value:keyword | hash:keyword
+test          | 9f86d081884c7d659a2feaa0c55ad015a3bf4f1b2b0b822cd15d6c15b0f00a08
+;
```

Comment:
Could you write an example that pulls from a table? It's mostly out of paranoia. The unit tests cover this case, but these run so fast that it's not a big cost to add one or two here as well.